elements/pfe-cta/demo/demo.css

```diff
@@ -18,12 +18,10 @@ pfe-band[pfe-color="lightest"] pfe-card[pfe-color="lightest"] {
 }
 
 .resize {
-  display: flex;
-  flex-flow: row nowrap;
   resize: horizontal;
```

Comment:
This was causing the focus state to look wonky 
![Screen Shot 2020-03-18 at 4 20 40 PM](https://user-images.githubusercontent.com/456863/77007661-f81ea900-693a-11ea-9989-f3bed9f89fc7.png)